docs/spec/spec.md

```diff
@@ -323,6 +323,7 @@ spec:  # One of "runLatest", "release", "pinned" (DEPRECATED), or "manual"
   release:
     # Ordered list of 1 or 2 revisions. First revision is traffic target
     # "current" and second revision is traffic target "candidate".
+    # It is possible to specify `"@latestRevision"` string as a shortcut to the lastest available revision.
```

Comment:
I'd probably go with `@latest`

Reply:
That was the latest in the document. :-)
I am fine either way.

Reply:
Done